todo_app/app.py

```diff
@@ -1,13 +1,20 @@
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-from todo_app.data.session_items import add_item, get_items
+from todo_app.data.trello_items import add_item, complete_item, get_items
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 @app.route('/add', methods=['POST'])
-def submitFilm():
-    add_item(request.form.get('title'))
+def addToDo():
+    print(request.form.get('new-item'))
+    add_item(request.form.get('new-item'))
+    return redirect('/')
+
+@app.route('/complete/<id>', methods=['GET'])
```

Comment:
Using Complete & Remove interchangeably threw me for a moment, but I guess that's a reasonable assumption at this point! We will assume in future that those are two separate functions (i.e. we might still want to see a list of completed tasks) so just one to be aware of 🙂

Reply:
Separately though, even if this is halfway between an update and a delete (i.e. it updates Trello, seemingly deletes from our UI), I think it's still nicer to have it as a POST request than a GET request